lib/referee-sinon.js

```diff
@@ -38,10 +39,18 @@ sinon.expectation.fail = function (message) {
 // Lazy bind the format method to referee's. This way, Sinon will
 // always format objects like referee does, even if referee is
 // configured after referee-sinon is loaded
-sinon.format = function () {
-    return referee.format.apply(referee, arguments);
+var formatter = formatio.configure({
+    quoteStrings: false,
+    limitChildrenCount: 250
+});
+
+var format = function () {
```

Comment:
Is there any particular reason for making this a [function expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function) and not a [function declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)?

In any case, it would be good to name the function, so that some of the less capable runtimes don't just display `(anonymous)` in stack traces.

Reply:
The particular reason was that it required less keystrokes in vim to make the change 😆 I've added a commit to make it a function declaration. Btw it could also be a "named function expression" which is still an expression but also shows the name in stacks `</smartypants>`.

Reply:
I really should get round to writing that blog post about all the different ways of creating functions in JavaScript ...